apps/functions/document-check/check-document/back-office-api-client.js

```diff
@@ -16,6 +16,40 @@ const sendRequestToBackOffice = async (documentGuid, machineAction) => {
 		.json();
 };
 
+/**
+ *
+ * @param {string} documentGuid
+ * @returns {Promise<{ guid: string, documentName: string, fromFrontOffice: boolean } | null>}
+ * */
+export const getDocumentProperties = async (documentGuid) => {
+	try {
+		const result = await got
+			.get(`https://${config.API_HOST}/applications/document/${documentGuid}/properties`)
+			.json();
+
+		return result;
+	} catch {
+		return null;
```

Comment:
Maybe this is OK, but it could hide an error that we need to see. Perhaps log it?